File: modules/angular2/src/core/platform_bindings.ts
import {provide} from 'angular2/src/core/di';
import {Provider} from "angular2/src/core/di/provider";
import {ExceptionHandler} from 'angular2/src/facade/exceptions';
import {DOM} from 'angular2/src/core/dom/dom_adapter';

export const EXCEPTION_PROVIDER: Provider =
    provide(ExceptionHandler, {useFactory: () => new ExceptionHandler(DOM, false), deps: []});

export const EXCEPTION_BINDING = EXCEPTION_PROVIDER;

File: modules/angular2/src/core/platform_bindings.dart
library angular2.src.core.platform_bindings;


import 'package:angular2/core.dart';
import 'package:angular2/src/core/facade/exceptions.dart';
import 'package:angular2/src/core/dom/dom_adapter.dart';

exceptionFactory() => new ExceptionHandler(DOM, true);

const EXCEPTION_PROVIDER = const Binding(ExceptionHandler, toFactory: exceptionFactory, deps: const []);

const EXCEPTION_BINDING = EXCEPTION_PROVIDER;
